checkov/github/checks/webhooks_https_orgs.py

```diff
@@ -11,7 +11,7 @@
 
 class WebhookHttpsOrg(BaseGithubCheck):
     def __init__(self) -> None:
-        name = "Ensure GitHub organization webhooks are using HTTPS"
+        name = "Ensure GitHub organization webhooks are using HTTPS - CIS 4.3.4 (CKV_GITHUB_7 is the other one)"
```

Comment:
```suggestion
        name = "Ensure GitHub organization webhooks are using HTTPS - CIS 4.3.4"
```
no need for the last part